core/src/test/scala/com/mesosphere/usi/core/helpers/FakeMesos.scala

```diff
@@ -0,0 +1,61 @@
+package com.mesosphere.usi.core.helpers
+
+import akka.NotUsed
+import akka.stream.scaladsl.Flow
+import com.mesosphere.usi.core.models.AgentId
+import com.mesosphere.usi.core.protos.ProtoBuilders
+import org.apache.mesos.v1.{Protos => Mesos}
+import org.apache.mesos.v1.scheduler.Protos.{Call => MesosCall, Event => MesosEvent}
+import scala.collection.JavaConverters._
+
+object FakeMesos {
+  import com.mesosphere.usi.core.protos.ProtoBuilders._
+  import com.mesosphere.usi.core.protos.ProtoConversions._
+  val fakeAgentId = AgentId("fake-agent")
```

Comment:
and `mockAgentId`? and `mock-agent`? and `mockMesosLoggingFlow`? and... you get the idea ;)

Reply:
lol maybe later hahaha lets merge this already